addons/website/views/snippets/snippets.xml

```diff
@@ -699,7 +699,7 @@
          data-no-check="true"
          groups="website.group_website_designer">
 
-        <we-select string="Scroll Effect" data-dependencies="!header_sidebar_opt" class="o_scroll_effects_selector">
+        <we-select string="Scroll Effect" data-dependencies="!header_sidebar_opt" class="o_scroll_effects_selector" data-variable="header-scroll-effect">
```

Comment:
> you will have to suppose you cannot know the value of that variable 😬

mmm.. I think that I have already  done it here.
https://github.com/odoo/odoo/pull/98558/files/71d01a1b1b0e013b3b718dbe65686aebaa8eec55#diff-e7fd12b506f935d1a853ecff44d3fedf867796ddef908e32fd15b5da672a851bR196

If the variable doesn't exist it fallbacks "gracefully "... is this what you mean?

Reply:
> If the variable doesn't exist it fallbacks "gracefully "... is this what you mean?

Difficult to say without testing. Does it mean that with your implementation if the value is unknown ("null"), you have an imperfect-but-not-broken behavior? Not ideal but acceptable I guess :)

Reply:
Yes. There are 2 scenarios:

- You are on a v16 db and you use default templates: all fine 👍 

- You're in an old db OR you use a custom header:
chances are that the sidebar "will stick" farther than necessary, but at least it will not be covered by the navbar. In this case you can use the CSS variable to adjust the value.